<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPersonasContactoCliente</class>
 <widget class="QDialog" name="frmPersonasContactoCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personas de contacto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../terra.qrc">
    <normaloff>:/Icons/PNG/terra.png</normaloff>:/Icons/PNG/terra.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#4c6088</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(175,195,225);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="btnEliminar">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Eliminar seleccionado</string>
        </property>
        <property name="icon">
         <iconset resource="../terra.qrc">
          <normaloff>:/Icons/PNG/Delete.png</normaloff>:/Icons/PNG/Delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tabla_PersonasContacto">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(175,195,225);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="txtNombre">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Descripción T1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtDescripcionT1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Descripción T2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="txtDescripcionT2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Descripción T3:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="txtDescripcionT3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Telefono 1:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtTelefono1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>telefono 2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="txtTelefono2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Telefono 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="txtTelefono3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Descripción M1:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtDescripcionM1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Descripción M2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="txtDescripcionM2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Móvil 1:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtMovil1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Móvil 2:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="txtMovil2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="btnAnadir">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Añadir</string>
        </property>
        <property name="icon">
         <iconset resource="../terra.qrc">
          <normaloff>:/Icons/PNG/add.png</normaloff>:/Icons/PNG/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>648</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnAceptar">
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>33</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:disabled{color: 0; }
QPushButton:enabled{color: rgb(255,255,255); }</string>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
     <property name="icon">
      <iconset resource="../terra.qrc">
       <normaloff>:/Icons/PNG/OK.png</normaloff>:/Icons/PNG/OK.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtDescripcionT1</tabstop>
  <tabstop>txtTelefono1</tabstop>
  <tabstop>txtDescripcionT2</tabstop>
  <tabstop>txtTelefono2</tabstop>
  <tabstop>txtDescripcionT3</tabstop>
  <tabstop>txtTelefono3</tabstop>
  <tabstop>txtDescripcionM1</tabstop>
  <tabstop>txtMovil1</tabstop>
  <tabstop>txtDescripcionM2</tabstop>
  <tabstop>txtMovil2</tabstop>
  <tabstop>btnAnadir</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>tabla_PersonasContacto</tabstop>
  <tabstop>btnEliminar</tabstop>
 </tabstops>
 <resources>
  <include location="../terra.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>frmPersonasContactoCliente</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAnadir</sender>
   <signal>clicked()</signal>
   <receiver>frmPersonasContactoCliente</receiver>
   <slot>Anadir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEliminar</sender>
   <signal>clicked()</signal>
   <receiver>frmPersonasContactoCliente</receiver>
   <slot>Borrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Anadir()</slot>
  <slot>Borrar()</slot>
 </slots>
</ui>
